core-bundle/contao/drivers/DC_Table.php

```diff
@@ -4805,7 +4757,7 @@ protected function listView()
 		}
 
 		$parameters['message'] = Message::generate();
-		$parameters['operations'] = $operations;
+		$parameters['global_operations'] = $operations;
```

Comment:
Was that a bug? (seems unrelated to this PR)

Reply:
It is related. It was an inconsistency which got obvious when refactoring the templates (see the template, that also changed and now uses a common parent).